subsys/net/lib/nrf_cloud/src/nrf_cloud_fota.c

```diff
@@ -171,6 +173,53 @@ static int fota_settings_set(const char *key, size_t len_rd,
 	return 0;
 }
 
+static int load_fota_settings(void)
+{
+	int ret = settings_subsys_init();
+
+	if (ret) {
+		LOG_ERR("Settings init failed: %d", ret);
+	}
+
+	ret = settings_load_subtree(settings_handler_fota.name);
+	if (ret) {
+		LOG_ERR("Cannot load settings: %d", ret);
+	}
+
+	return ret;
+}
+
+int nrf_cloud_modem_fota_pending_job_process(const int modem_lib_init_result)
+{
+	/* Nothing to do if modem lib did not indicate a @ref nrf_modem_dfu result */
+	if (modem_lib_init_result == 0) {
+		return 0;
+	}
+
+	if (settings_subsys_init()) {
```

Comment:
`settings_subsys_init()` is called twice -- even if idempotent, is that necessary?